src/cdn_definitions/schema.json

```diff
@@ -29,15 +147,169 @@
             },
             "type": "array",
             "uniqueItems": true
+        },
+        "product_id": {
+            "pattern": "^[0-9]+$",
+            "type": "string"
+        },
+        "product_id_list": {
+            "items": {
+                "$ref": "#/definitions/product_id"
+            },
+            "type": "array",
+            "uniqueItems": true
+        },
+        "release_stream_mapping": {
+            "additionalProperties": false,
+            "patternProperties": {
+                ".*": {
+                    "$ref": "#/definitions/release_version_list"
+                }
+            },
+            "properties": {
+                "aus": {
+                    "$ref": "#/definitions/release_version_list"
+                },
+                "e4s": {
+                    "$ref": "#/definitions/release_version_list"
+                },
+                "eus": {
+                    "$ref": "#/definitions/release_version_list"
+                },
+                "tus": {
+                    "$ref": "#/definitions/release_version_list"
+                }
+            },
+            "requiredProperties": [
+                "aus",
+                "eus",
+                "tus",
+                "e4s"
+            ],
+            "type": "object"
+        },
+        "release_version": {
+            "pattern": "^[0-9]+\\.[0-9]+$",
+            "type": "string"
+        },
+        "release_version_list": {
+            "items": {
+                "$ref": "#/definitions/release_version"
+            },
+            "type": "array",
+            "uniqueItems": true
+        },
+        "signing_key": {
+            "pattern": "^[A-Fa-f0-9]{8}",
```

Comment:
I don't think we should support both A-F and a-f, can we pick one or the other?

I wasn't sure about it but then I saw below in signing_key_list that you put uniqueItems: true.  That doesn't provide much value if, for example, "abcdef00" and "ABCDEF00" are both allowed to be in the list.